JS/Lesson17/onder/script.js

```diff
@@ -134,19 +134,54 @@ console.log('----------------------------');
 // Input: ({name: "John", age: 30}, "city", "New York")
 // Output: {name: "John", age: 30, city: "New York"}
 
+//I'm really not sure about 10-11-12-13 questions.
+const object = { name: "John", age: 30 };
+const key = "city";
```

Comment:
Check this suggestion: https://github.com/frontend-from-0/240123-exercises/pull/50#discussion_r1307930036